### DISCUSSION TITLE
我的windows系统无法将lantern设置系统代理

### DISCUSSION THREAD

**Discussion by 939405780** [getlantern/lantern-forum-cn#707] [Q&A]
打开lantern后，lantern页面提示：“无法访问此网站localhost 拒绝了我们的连接请求。”
lantern的logs文件里有报错，如下：
error_location=github.com/getlantern/lantern-desktop/loconf.get (loconf.go:81) error_op=Get error_text=dial tcp: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found. error_type=url.Error]
ERROR loconfer: loconfscanner.go:48   at github.com/getlantern/lantern-desktop/loconf.get (loconf.go:81)
ERROR loconfer: loconfscanner.go:48   at github.com/getlantern/lantern-desktop/loconf.Get (loconf.go:70)
ERROR loconfer: loconfscanner.go:48   at github.com/getlantern/lantern-desktop/desktop.(*loconfer).scan.func1 (loconfscanner.go:46)
ERROR loconfer: loconfscanner.go:48   at github.com/getlantern/lantern-desktop/desktop.(*loconfer).scan.func2 (loconfscanner.go:60)
ERROR loconfer: loconfscanner.go:48   at runtime.goexit (asm_386.s:1363)
ERROR loconfer: loconfscanner.go:48 Caused by: dial tcp: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found.
ERROR loconfer: loconfscanner.go:48 Caused by: lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found.
moltres-2d37f0d3-3f66-4661-a0ba-42cf2b452a72 failed: handshake failed: proxied handshake failed: read tcp 192.168.5.2:53863->158.179.170.121:443: wsarecv: An existing connection was forcibly closed by the remote host.
panic: unaligned 64-bit atomic operation

**Comment by MaxMKMax** [comment]
@939405780 谢谢您的反馈. 我已经帮您开ticket了. 请您放心, 我们会尽快解决问题. 请问您有log号码吗? 也提供给我一下. 谢谢!

**Comment by MaxMKMax** [comment]
@939405780 您好. 非常抱歉影响了您的体验感. 您那里能帮我提交报告然后给我 8 票号.吗? 如果应用闪退, 提交不了, 您可以手动帮我找log然后提供给我吗? 非常感谢! 有log我们就能马上修复了. 麻烦您帮忙了. 谢谢! 
可以是看这么找:
C:\Users\username\AppData\Roaming\Lantern

各位放心. 我已经把你们的问题提交给技术团队了. 我们会尽快修复的. 谢谢您的耐心等待. 也很抱歉这几天影响了你们的体验感. 谢谢你们的反馈也感谢你们支持蓝灯!

**Reply by xiaomihaha** [reply]
[lantern.log](https://github.com/user-attachments/files/16199912/lantern.log)
